core/scripts/chaincli/command/keeper/registry.go

```diff
@@ -51,6 +58,7 @@ var withdrawFromRegistryCmd = &cobra.Command{
 }
 
 func init() {
+	deployRegistryCmd.Flags().BoolP("verify", "v", false, "Specify if contracts should be verified on Etherscan")
```

Comment:
Does it make sense to replace "Etherscan" with something more generic like "the block explorer"?

Reply:
Good point. With this PR, we support only etherscan. Adding other explorers would require another PR, including changes here. I think it is better to be specific in the message?